src/views/InternalApp/TheBets/TheBets.vue

```diff
@@ -79,6 +90,12 @@ onMounted(async () => {
         :teamB="bet.teamB"
         @selectBet="selectBet" />
     </div>
+    <CrushPagination 
+      :currentPage="betStore.currentPage"
+      :totalPages="betStore.totalPages || 0"
+      @changePage="changePage"
+      class="create-container-pagination"
```

Comment:
Please follow the code style guide 
https://app.clickup.com/14221792/v/dc/dj0f0-2765/dj0f0-225